app/models/solr_object.rb

```diff
@@ -0,0 +1,18 @@
+require 'json'
+require 'net/http'
+
+class SolrObject < BaseHttpObject 
+  def initialize()
+    @config = Rails.application.config_for(:vudl)
+  end
+
+  def query_url
+    URI(@config["solr_query_url"])
```

Comment:
I think we should add:

`  solr_query_url: http://myserver:8983/solr/core/query`

to vudl.yml as an example configuration in parallel with adding this file, just so it's more clear that this configuration is now needed. I suspect that we put a real URL in your local vudl.yml but forgot to set up that change for this generic PR.

Reply:
sounds good!